.github/workflows/static.yml

```diff
@@ -68,3 +68,33 @@ jobs:
       - name: Run golangci-lint
         run: |
           nix run .#scripts.golangci-lint
+
+  clean-patches:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 15
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          path: contrast
+      - uses: ./contrast/.github/actions/setup_nix
+        with:
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          cachixToken: ${{ secrets.CACHIX_AUTH_TOKEN }}
+      - id: kataversion
+        working-directory: contrast
+        run: |
+          echo "v=$(nix eval --raw .#kata.kata-runtime.version)" >> "$GITHUB_OUTPUT"
+      - name: Check out Kata
+        run: |
+          git clone --depth 1 -b "${{ steps.kataversion.outputs.v }}" https://github.com/kata-containers/kata-containers.git kata
+      - name: Apply patches
+        working-directory: kata
+        run: |
+          git config --global user.email "actionsbot@example.invalid"
+          git config --global user.name "Actions Bot"
+          git am --no-3way --whitespace=fix ../contrast/packages/by-name/kata/kata-runtime/*.patch
```

Comment:
Why is this command so different from what we document in the patching guide?

Reply:
* `--no-3way`: If the patch workflow is thoroughly followed, 3-way merges should not be necessary. My idea of adding the flag explicitly here was to catch unclean patches in CI. If the existing patches are clean, it does not make a difference to the developer workflow, though, but I can remove the `-3` from the docs to align the two.
* `--whitespace=fix`: This should catch newly introduced whitespace "problems". However, not everybody subscribes to the whitespace dogma of git, so I would not mind removing it. wdyt?

Reply:
I don't really care what we enforce here, but I guess we should align somehow so we don't have the ci failing even everything was done right. Also this isn't using `/0*.patch` to filter out the other patches, is it.

Reply:
Ok, I'm going to
* remove the whitespace check, as its more likely to be annoying than helpful
* change the docs not to use `-3` and explain why

> Also this isn't using /0*.patch to filter out the other patches.

Good catch, thanks!

Reply:
Addressed.